### ISSUE TITLE
Unable to use env/build variables in Json Reports Path field

### ISSUE THREAD

**Issue by ghost** [damianszczepanik/cucumber-reporting-jenkins#66]
I pass parameter "folder" through "This build is parameterized", and I want to use it in Json Reports Path field, e. g. target/${folder}.
But I get: java.lang.IllegalStateException: basedir D:\project\target\${folder} does not exist.

I tried: target/${folder} and target/$folder
Also tried to define env variable, but it still couldn't find folder: D:\project\target\${MY_VAR}, D:\project\target\$MY_VAR, D:\project\target\%MY_VAR%

**Comment by damianszczepanik** [comment]
I need json file to investigate this problem

**Comment by srohan** [comment]
Was this a plugin issue ? I see that a bug was added on Aug 22, 2015 and was closed on Dec 15, 2015. Do we need to update the plugin to use environment variables in report path or is there any other solution/work around for this problem. Appreciate your help !

**Comment by damianszczepanik** [comment]
I did not get feedback so issue was closed

**Comment by srohan** [comment]
ahh, I see, thanks for the response. So let me rephrase my question - would passing an env variable to cucumber report path be fixed in the near future ? or it is not even an issue of the plugin at the first place ?

**Comment by srohan** [comment]
Just checking, please could you let me know if you have any updates on this ? Thanks !

**Comment by noel-yap** [comment]
@damianszczepanik , can you elaborate on what you need in order to implement this feature?
